soroban-sdk/src/env.rs

```diff
@@ -469,6 +469,79 @@ use std::{path::Path, rc::Rc};
 #[cfg(any(test, feature = "testutils"))]
 use xdr::{LedgerEntry, LedgerKey, LedgerKeyContractData, SorobanAuthorizationEntry};
 
+#[cfg(any(test, feature = "testutils"))]
+pub struct TestStellarAssetContractUtil {
+    pub address: Address,
+    env: Env,
+    issuer_id: xdr::AccountId,
+}
+
+#[cfg(any(test, feature = "testutils"))]
+impl TestStellarAssetContractUtil {
```

Comment:
Suggested name change, since util is so broad, and the utilities seemed to be focused on interacting with the issuer.
```suggestion
impl TestStellarAssetContractIssuer {
```

Reply:
I'm now using `StellarAssetContract` and `StellarAssetIssuer`.